src/server.ts

```diff
@@ -34,15 +34,10 @@ const clientLib = path.resolve(__dirname, '..', 'client', 'lib');
 export class Server {
   readonly url: string;
   private readonly server: net.Server;
-
-  // Even though we're running benchmarks in series, we give each run an id and
-  // make sure that we associate result messages with the correct run. This
-  // prevents any spurrious race conditions and enables one runner to launch
-  // multiple clients eventually.
-  private currentRunId = 0;
```

Comment:
+1 to dropping this for YAGNI. We might some day want to do concurrent runs via sauce or whatnot, but that's a problem for then